SD_Card_Driver/Core/Src/LOS_D_sd.cpp

```diff
@@ -0,0 +1,96 @@
+#include "LOS_D_sd.hpp"
+
+int SDCard::init()

```

Comment:
Can we just use the constructor for this instead of an `init()` method?

Reply:
Yup. We can probably also use the destructor to dismount FatFS instead of doing it in the `cleanup` method.